library/private/options_tools.h.in

```diff
@@ -0,0 +1,263 @@
+#ifndef f3d_options_tools_h
+#define f3d_options_tools_h
+
+#include "options.h"
+#include "types.h"
+
+#include <sstream>
+
+namespace f3d
+{
+namespace options_tools
+{
+
+// TODO expose parse methods in options API ?
+//----------------------------------------------------------------------------
+/**
+ * Parse provided string into provided boolean reference
+ * Numerical and boolaplha are both supported
+ */
+static void parse(const std::string& str, bool& value)
```

Comment:
as per discussion on discord, please try to return the result instead of passing a reference in arguments.